tests/api_tests/nodes/test_views.py

```diff
@@ -486,7 +526,7 @@ def test_update_public_project_logged_in(self):
         assert_equal(res.status_code, 403)
 
     def test_update_private_project_logged_out(self):
-        res = self.app.put_json(self.private_url, {
+        res = self.app.put(self.private_url, {
```

Comment:
What do you mean by a possible regression?  I meant to flag this for you, and forgot.  I have no idea why put_json is not working.  I reverted to put, post, etc. for the time being because it seemed like the tests were inconsistent anyway.